shopgate/pwa
# Refactored the FormBuilder to support replacing of existing and adding custom elements. Also added class names to form elements for advanced component styling.

# Description

Refactored the FormBuilder to support replacing of existing and adding custom elements. Also added class names to form elements for advanced component styling.

## Type of change


## How to test it

Check if the form elements still work as they did before. Then create a new page and instantiate the form builder to create a custom form. All form element components should have an own classname assigned to them.
